docs/guide/actions.md

```diff
@@ -4,7 +4,7 @@ Actions are allow-listed functions with a string syntax you can embed in links a
 
 ## Action methods
 
-Action methods are methods on your app or widgets prefixed with `action_`. Aside from the prefix these are regular methods which you could call directly if you wished.
+Actions are handled by _action methods_ of your app or widgets - methods whose names are prefixed with `action_`. These are regular methods which you can call directly, but normally they are triggered by actions.
```

Comment:
This change is partly pedantry and partly a wish to add more context, but "aside from the prefix" seems completely redundant. It's quite probably a matter of taste and style preference.